client/src/graphql/mutations/addConditionToInputGroup.graphql

```diff
@@ -0,0 +1,22 @@
+#import "../fragments/entireCondition.graphql"
+
+mutation addConditionToInputGroup(
+  $inputGroupId: ID!
+  $action: String
```

Comment:
isn't this required ? as well as the other fields ?

Reply:
I first create empty conditions and fill them after. So we could even remove these arguments for the addConditionToInputGroup mutation. What do you think?